app_llm_test.go

```
@@ -68,6 +72,7 @@ func TestPromptTokenLimits(t *testing.T) {
 	require.NoError(t, err)
 	correspondentTemplate, err = template.New("correspondent").Parse(testCorrespondentTemplate)
 	require.NoError(t, err)
+	createdDateTemplate, err = template.New("created_date").Parse(testCreatedDateContentTemplate)
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)